packages/sku/src/utils/polyfillRegistry.ts

```diff
@@ -0,0 +1,42 @@
+export interface PolyfillRegistryEntry {
+  reason: string;
+  docsUrl?: string;
+}
+
+export type PolyfillRegistry = Record<string, PolyfillRegistryEntry>;
+
+/**
+ * Registry of polyfills that are unnecessary for sku's supported browsers
+ */
+export const POLYFILL_REGISTRY: PolyfillRegistry = {
+  'core-js': {
+    reason:
+      'Most ES2015-ES2017 features are natively supported in modern browsers',
```

Comment:
`core-js` contains polyfills for a lot more than just ES2015-ES2017 features. We might need to either be more specific about which polyfills we target, or maybe put a blanket message that says "`core-js` polyfills lots of behaviour that you may or may not use. Consider either removing it or using targeted polyfill for specific APIs".